spec/features/features_spec.rb

```diff
@@ -0,0 +1,94 @@
+require 'takeaway'
+require 'menu'
+require 'menu_item'
+require 'order'
+require 'sms_sender'
+
+describe 'User stories' do
+  let(:menu) { Menu.new }
+  let(:order) { Order.new }
+  let(:sms_sender) { SMSSender.new }
+  # using this uses a real instance of SMSSender, sending real texts
+  let(:takeaway) { Takeaway.new(menu, order, sms_sender) }
+
+  # As a customer
+  # So that I can check if I want to order something
+  # I would like to see a list of dishes with prices
+    
+    # heredoc -> to define multiline strings
+    # used to compare against output from initialised menu
+  dummy_menu = <<-HEREDOC
+1. Pizza: £8.00
+2. Burger: £9.00
+3. Chips: £4.50
+  HEREDOC
+  
```

Comment:
I don't quite get how the dummy menu defined above can be output by your Takeaway, without the dummy meny being injected at some point...